econml/_ortho_learner.py

```diff
@@ -1035,26 +1083,45 @@ def score(self, Y, T, X=None, W=None, Z=None, sample_weight=None, groups=None):
         if self.discrete_outcome:
             Y = self.outcome_transformer.transform(Y).reshape(-1, 1)
         n_iters = len(self._models_nuisance)
-        n_splits = len(self._models_nuisance[0])
+        # self._models_nuisance will be a list of lists or a list of list of lists
+        # so we use self._ortho_learner_model_nuisance to determine the nesting level
+        expand_list = not isinstance(self._ortho_learner_model_nuisance, list)
+        if expand_list:
+            n_selectors = 1
+            n_splits = len(self._models_nuisance[0])
+        else:
+            n_selectors = len(self._models_nuisance[0])
+            n_splits = len(self._models_nuisance[0][1])
```

Comment:
I'm having trouble understanding the 1 index here. Is it arbitrary? i.e. could the 1 index be replaced with a 0 index and also work fine?

Reply:
Yes, I'll change to 0 - there should be the same number of folds for each nuisance model so it's arbitrary, and in pratice DRIV is the only set of estimators where we have more than one nuisance model so 1 works okay, but 0 is a more natural choice.